network/wsNetwork.go

```diff
@@ -1591,7 +1740,25 @@ func (wn *WebsocketNetwork) tryConnect(addr, gossipAddr string) {
 		return
 	}
 
-	peer := &wsPeer{wsPeerCore: wsPeerCore{net: wn, rootURL: addr}, conn: conn, outgoing: true, incomingMsgFilter: wn.incomingMsgFilter, createTime: time.Now()}
+	throttledConnection := false
```

Comment:
wonder if there is a code pass when a connection is dropped but removePeer is not called so that throttledOutgoingConnections counter can be screwed

Reply:
seems like we are clear

Reply:
It would be great if you can find such. If you do, please let me know, since it also means that we didn't remove the entry from the peers list.